tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::wp_is_numeric_array
+ */
+class Tests_Functions__wp_check_existing_file_names extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Functions__WpCheckExistingFileNames extends WP_UnitTestCase {
```
- Corrected test classname.